src/app/views.py

```diff
@@ -1840,22 +1840,17 @@ def post_to_github(request):
         return HttpResponse(dumps({"message": "User should be logged in to use this feature"}), status=400)
 
 def handler400(request, exception=None):
-    return render_to_response('app/400.html',
-        context_instance = RequestContext(request)
-    )
+    context = {}
+    return render(request, 'app/400.html', context)
 
 def handler403(request, exception=None):
-    return render_to_response('app/403.html',
-        context_instance = RequestContext(request)
-    )
+    context = {}
+    return render(request, 'app/403.html', context)
 
 def handler404(request, exception=None):
-    return render_to_response('app/404.html',
-        context_instance = RequestContext(request),
-        status=404
-    )
+    context = {}
+    return render(request, 'app/404.html', context)
 
 def handler500(request, exception=None):
-    return render_to_response('app/500.html',
-        context_instance = RequestContext(request)
-    )
+    context = {}
+    return render(request, 'app/500.html', context)
```

Comment:
@Ugtan Please comment on these changes - either in the code or on github PR? Like why was this changed. Like for this one - `render_to_response` shortcut was deprecated in Django 2.0, and is removed in Django 3.0.

Reply:
Alright will do.